subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -1377,8 +1470,83 @@ uint32_t radio_tmr_start_tick(uint8_t trx, uint32_t tick)
 	remainder_us = 1;
 	nrf_timer_cc_set(EVENT_TIMER, 0, remainder_us);
 
-	nrf_rtc_cc_set(NRF_RTC, 2, tick);
+#if defined(CONFIG_BT_CTLR_NRF_GRTC)
+	uint32_t cntr_l, cntr_h, cntr_h_overflow, stale;
+
+	/* Disable capture/compare */
```

Comment:
* Same comments as before for the equivalent chunk in `radio_tmr_start()`

* Given that it is so similar to the code in `radio_tmr_start()` should all this not be refactored into a common function?